src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form-big"
```

Comment:
```suggestion
      class="form"
```
for small form set classes `form form-small`  and reset only styles which changed in small form.
same for oher elelments

![Screenshot from 2023-08-18 12-46-18](https://github.com/mate-academy/layout_search-bar-airbnb/assets/33029864/3b24ed26-a30c-4cda-9a03-d1be0e641b73)

Reply:
fixed